packages/api/src/car.js

```diff
@@ -129,27 +121,36 @@ export async function carGet (request, env, ctx) {
  * @param {import('./index').Ctx} ctx
  */
 export async function carPost (request, env, ctx) {
+  let blob = await request.blob()
+  // Ensure car blob.type is set; it is used by the cluster client to set the foramt=car flag on the /add call.
+  blob = blob.slice(0, blob.size, 'application/car')
```

Comment:
Given we will send CAR files to cluster either way, shouldn't this live in `handleCarUpload`

Reply:
I _think_ you can just do:

```js
cluster.add(blob, { format: 'car' })
```

...and avoid this dance.

https://github.com/nftstorage/ipfs-cluster/blob/0458e04c0b0331fab4f08f0c092b774c71c0ead2/index.js#L41